sdk/storage/azure-storage-file-datalake/out/azure/storage/filedatalake/_models.pyi

```diff
@@ -93,10 +93,14 @@ class DirectorySasPermissions(BlobSasPermissions):
 class FileSasPermissions(BlobSasPermissions):
     def __init__(self, read: bool = ..., create: bool = ..., write: bool = ..., delete: bool = ...) -> None: ...
 
+class AccessPolicy(BlobAccessPolicy):
+    def __init__(self, permission: Optional[Any] = ..., expiry: Optional[Any] = ..., start: Optional[Any] = ...) -> None: ...
```

Comment:
The order between expiry and start seem odd. And, semi-related, should these be kwargs since there is not necessarily a well defined order between them?

Reply:
I agree the order should be reversed! While I don't think we want to put it in kwargs. it's kinda like to specify a start and expiry date for sas token. If you set the AccessPolicy on the file system, when you generate sas token you don't have to specify the start and expiry date as long as you specify the policy id you've created.

Reply:
```
def __init__(self, permission=none, expiry=None, **kwargs)
```
In Python3:
```
def __init__(self, permission=None, expiry=None, *, start=None)
```